.rubocop_todo.yml

```diff
@@ -42,7 +41,19 @@ Lint/MissingSuper:
     - 'lib/chewy/type/adapter/object.rb'
     - 'lib/chewy/type/adapter/orm.rb'
 
-# Offense count: 36
+# Offense count: 3
+Lint/SymbolConversion:
```

Comment:
@AlexVPopov I removed all the new offences. And I added `Ruby 2.7` as target version to `rubocop.yml` to avoid Rubocop CI error https://github.com/toptal/chewy/runs/2235043027?check_suite_focus=true . 

Then I asked in #chewy chat, is it okay. So, if adding target version is okay, PR will be ready for your second check :)